src/Metadata/Resource/ResourceMetadataCollection.php

```diff
@@ -102,7 +102,7 @@ public function getOperation(?string $operationName = null, bool $forceCollectio
     private function handleNotFound(string $operationName, ?ApiResource $metadata): void
     {
         // Hide the FQDN in the exception message if possible
-        $shortName = $metadata ? $metadata->getShortName() : $this->resourceClass;
+        $shortName = $metadata?->getShortName() ? $metadata->getShortName() : $this->resourceClass;
```

Comment:
Why this change?

Reply:
`getShortName` can return null, if we don't check the method it happens that we don't have the class that is impacted by the error because the shortName is empty